net.sourceforge.GrandOrgue.yml

```diff
@@ -0,0 +1,69 @@
+app-id: net.sourceforge.GrandOrgue
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: GrandOrgue
+rename-desktop-file: GrandOrgue.desktop
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --filesystem=home
+  - --socket=system-bus
+  - --filesystem=xdg-run/pipewire-0
```

Comment:
I assume this is needed for jack?

Reply:
Yes. It's documented in the next commit. Thanks!